testdata/config/metrics.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: prometheus
+metadata:
+  name: prometheus
+  namespace: istio-config-default
+spec:
+  metrics:
+  - name: requestcount
+    kind: COUNTER
+    label_names: [ source, target, service, method, version,  response_code ]
+  - name: requestduration
+    kind: COUNTER
+    label_names: [ source, target, service, method, version,  response_code ]
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: metric
+metadata:
+  name: requestcount
+  namespace: istio-config-default
+spec:
+  value: "1"
+  dimensions:
+    source: source.labels["app"] | "unknown"
+    target: target.service | "unknown"
+    service: target.labels["app"] | "unknown"
+    method: request.path | "unknown"
+    version: target.labels["version"] | "unknown"
+    response_code: response.code | 200
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: metric
+metadata:
+  name: requestduration
+  namespace: istio-config-default
+spec:
+  value: response.latency | response.duration | "0ms"
+  dimensions:
+    source: source.labels["app"] | "unknown"
+    target: target.service | "unknown"
+    service: target.labels["app"] | "unknown"
+    method: request.path | "unknown"
+    version: target.labels["version"] | "unknown"
+    response_code: response.code | 200
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: mixer-rule
+metadata:
+  name: prommetrics
```

Comment:
Should these names be more verbose...like prommetrics_rule. This name prommetrics by itself is confusing.. other names are fine

Reply:
its `kind` is already mixer-rule